packages/fxa-shared/package.json

```diff
@@ -58,7 +58,7 @@
     "accept-language": "^2.0.17",
     "ajv": "^6.10.2",
     "bluebird": "^3.5.5",
-    "celebrate": "^10.0.1",
+    "celebrate": "11.0.0",
```

Comment:
did you mean `11` or `^11`?

Reply:
11. I don't trust the carots in version numbers, I like to convert to this style whenever I update versions

Reply:
@meandavejustice Ah, I asked about this [a while ago](https://github.com/mozilla/fxa/pull/2016#issuecomment-517180157), and was told that the FxA team had decided to use carets in a past retro. If you'd like to change how the versioning is tracked in package.json files, maybe open a pull request with an ADR that outlines the pros and cons?

Reply:
I'll just move it back for now, I haven't run into any issues on fxa with it yet, just previously with testpilot and lockwise.

Reply:
Cool, sounds good. Not sure what's up with the test failures; trying another run.